Traceback (most recent call last):
  File "/home/jfhlewyee/anaconda3/envs/jb/lib/python3.13/site-packages/jupyter_cache/executors/utils.py", line 58, in single_nb_execution
    executenb(
    ~~~~~~~~~^
        nb,
        ^^^
    ...<4 lines>...
        **kwargs,
        ^^^^^^^^^
    )
    ^
  File "/home/jfhlewyee/anaconda3/envs/jb/lib/python3.13/site-packages/nbclient/client.py", line 1319, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^
  File "/home/jfhlewyee/anaconda3/envs/jb/lib/python3.13/site-packages/jupyter_core/utils/__init__.py", line 165, in wrapped
    return loop.run_until_complete(inner)
           ~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^
  File "/home/jfhlewyee/anaconda3/envs/jb/lib/python3.13/asyncio/base_events.py", line 720, in run_until_complete
    return future.result()
           ~~~~~~~~~~~~~^^
  File "/home/jfhlewyee/anaconda3/envs/jb/lib/python3.13/site-packages/nbclient/client.py", line 709, in async_execute
    await self.async_execute_cell(
        cell, index, execution_count=self.code_cells_executed + 1
    )
  File "/home/jfhlewyee/anaconda3/envs/jb/lib/python3.13/site-packages/nbclient/client.py", line 1062, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "/home/jfhlewyee/anaconda3/envs/jb/lib/python3.13/site-packages/nbclient/client.py", line 918, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
from myst_nb import glue
MAD_Slim = get_total_MAD_of_whole_set(data)
glue("MAD_Slim", "{:.2f}".format(MAD_Slim), display=False)
glue("benchmark", benchmark, display=False)
glue("nof", nof, display=False)
------------------

----- stdout -----
Warning: MAD value is too high for system 32 in set MB16-43 and was excluded
------------------

[0;31m---------------------------------------------------------------------------[0m
[0;31mTypeError[0m                                 Traceback (most recent call last)
Cell [0;32mIn[3], line 2[0m
[1;32m      1[0m [38;5;28;01mfrom[39;00m[38;5;250m [39m[38;5;21;01mmyst_nb[39;00m[38;5;250m [39m[38;5;28;01mimport[39;00m glue
[0;32m----> 2[0m MAD_Slim [38;5;241m=[39m [43mget_total_MAD_of_whole_set[49m[43m([49m[43mdata[49m[43m)[49m
[1;32m      3[0m glue([38;5;124m"[39m[38;5;124mMAD_Slim[39m[38;5;124m"[39m, [38;5;124m"[39m[38;5;132;01m{:.2f}[39;00m[38;5;124m"[39m[38;5;241m.[39mformat(MAD_Slim), display[38;5;241m=[39m[38;5;28;01mFalse[39;00m)
[1;32m      4[0m glue([38;5;124m"[39m[38;5;124mbenchmark[39m[38;5;124m"[39m, benchmark, display[38;5;241m=[39m[38;5;28;01mFalse[39;00m)

Cell [0;32mIn[2], line 6[0m, in [0;36mget_total_MAD_of_whole_set[0;34m(data)[0m
[1;32m      4[0m [38;5;28;01mfor[39;00m system [38;5;129;01min[39;00m data[[38;5;28mset[39m]:
[1;32m      5[0m     AD [38;5;241m=[39m data[[38;5;28mset[39m][system][[38;5;124m"[39m[38;5;124mAD[39m[38;5;124m"[39m]
[0;32m----> 6[0m     [38;5;28;01mif[39;00m([43mAD[49m[43m [49m[38;5;241;43m<[39;49m[43m [49m[38;5;241;43m100[39;49m):
[1;32m      7[0m         ADs[38;5;241m.[39mappend(data[[38;5;28mset[39m][system][[38;5;124m"[39m[38;5;124mAD[39m[38;5;124m"[39m])
[1;32m      8[0m     [38;5;28;01melse[39;00m:

[0;31mTypeError[0m: '<' not supported between instances of 'str' and 'int'

